Src/Public/Invoke-AsBuiltReport.Rubrik.CDM.ps1

```diff
@@ -0,0 +1,1142 @@
+function Invoke-AsBuiltReport.Rubrik.CDM {
+    <#
+    .SYNOPSIS  
+        PowerShell script to document the configuration of Rubrik CDM in Word/HTML/XML/Text formats
+    .DESCRIPTION
+        Documents the configuration of the Rubrik CDM in Word/HTML/XML/Text formats using PScribo.
+    .NOTES
+        Version:        1.0.0
```

Comment:
I'd suggest version 0.x.x if it's not quite ready for prime time.

Reply:
done and done - good point!